SSA/Core/MLIRSyntax/GenericParser.lean

```diff
@@ -487,6 +487,7 @@ macro_rules
    let opsList <- `([mlir_ops| $ops])
    `(Region.mk "entry" [] $opsList)
 
+-- TODO: see if we can hide the `[mlir_op| ` in the same way as we did here for regions
```

Comment:
Was this addressed in https://github.com/opencompl/ssa/pull/361?

Reply:
It's actually already addressed here, I've dropped the TODO